library/maven/artifacts.bzl

```diff
@@ -204,7 +204,8 @@ artifacts = {
     "org.mockito:mockito-core": "2.6.4",
     "org.openjdk.jmh:jmh-core": "1.23",
     "org.openjdk.jmh:jmh-generator-annprocess": "1.23",
-    "org.rocksdb:rocksdbjni": "6.8.1",
+    "org.rocksdb:rocksdbjni": "6.11.4",
+    "org.rocksdb:rocksdbjni-dev": "6.11.4",
```

Comment:
Given that we're outputing `org.rocksdb:rocksdbjni-dev`, should we rename the whole package to `rocksdbjni-dev`?

Reply:
`org.rocksdb` is the name chosen by the RocksDB team which fits with standard package naming guidelines (i.e. take a domain name and reverse it). I think I prefer `rocksdb` - note that the packages in Java code are also named `org.rocksdb`, and it is only intended that we import one JAR at a time.